io/plugins/eu.esdihumboldt.hale.io.xls/src/eu/esdihumboldt/hale/io/xls/AbstractAnalyseTable.java

```diff
@@ -56,7 +59,18 @@ protected void analyse(URI location, int sheetNum) throws Exception {
 		InputStream inp = new BufferedInputStream(location.toURL().openStream());
 
 		try {
-			Workbook wb = WorkbookFactory.create(inp);
+//			https://poi.apache.org/components/spreadsheet/quick-guide.html#FileInputStream
+			Workbook wb;
+			if (location.getPath().contains(".xls")) {
```

Comment:
Isn't `.xls` also contained in `.xlsx`? Or which case should the `else` cover?

Reply:
Ok, but `location.getPath().contains(".xls")` will also match "file.xlsx".
Maybe use `endsWith`? Maybe also in combination with converting it to lowercase first?

Reply:
> Isn't .xls also contained in .xlsx?

Oh, I got confused with the meaning of these two w.r.t excel. Yes, in coding world it is contained in it. Sorry. 

Pushed the changes.